setup.py

```diff
@@ -14,10 +14,10 @@ def has_ext_modules(self):
 
 setuptools.setup(
     name="tf-seal",
-    version="dev",
+    version="0.1.0-rc0",
     packages=setuptools.find_packages(),
     package_data={'tf_seal': []},
-    python_requires=">=3.5",
+    python_requires="==3.7",
```

Comment:
Should this be >=3.7?

Reply:
No, right now we're only building a tensorflow custom build for python 3.7 so we're constrained to that.